projects/buttons/fastener/main.py

```diff
@@ -439,7 +456,7 @@ def create(stream_slug='lou'):
         "items": [
         {
             "key": "startup-script-url",
-            "value": "https://raw.githubusercontent.com/lucidworks/streams/master/projects/buttons/fastener/scripts/start-button.sh"
+            "value": "https://raw.githubusercontent.com/sudosoup/streams/fusion-5-dev/projects/buttons/fastener/scripts/start-button.sh"
```

Comment:
This needs to change to point the correct branch and repo. Make this change absolutely last.